modules/base/humanitec-agent.tf

```diff
@@ -0,0 +1,81 @@
+# Installs the humanitec-agent into the cluster
+
+# More details https://developer.humanitec.com/integration-and-extensions/humanitec-agent/overview/
+
+resource "kubernetes_namespace" "agent-namespace" {
+  metadata {
+    labels = {
+      "app.kubernetes.io/name"     = "humanitec-operator"
+      "app.kubernetes.io/instance" = "humanitec-operator"
```

Comment:
Could we add `"pod-security.kubernetes.io/enforce" = "restricted"`, I tested it, we support it, let's show it?